frontend/src/stores/game.ts

```diff
@@ -12,6 +12,35 @@ export const useGameStore = defineStore('game', () => {
 
   let currentGame = ref<IGameSettings | null>(null);
 
+  const waitForInvitation = function () {
 
-  return { socket, currentGame }
+  }
+
+  const stopWaitingForInvitation = function () {
```

Comment:
This store is waiting for the modal ?

Reply:
The websocket is subcribed to the "gameInvitation" event. The role of this function would be to stop responding to these event in some cases, for example when we are in a game. We would then need to subscribe again after the game